src/page-wrapper.js

```diff
@@ -90,12 +90,13 @@ export default function PageWrapper(props) {
     <Fragment>
       <Helmet>
         <link rel="icon" href="/favicon.ico" />
-        <link rel="stylesheet" href="/style.css" />
-        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com/" />
+        <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin />
         <link
           href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500&display=swap"
           rel="stylesheet"
         />
+        <link rel="stylesheet" href="/style.css" />
```

Comment:
I also don't know if order matters but we're gonna make sure to put it appropriate here!